src/rpc/blockchain.cpp

```diff
@@ -1273,7 +1273,7 @@ RPCHelpMan getblockchaininfo()
         RPCResult{
             RPCResult::Type::OBJ, "", "",
             {
-                {RPCResult::Type::STR, "chain", "current network name (main, test, signet, regtest)"},
+                {RPCResult::Type::STR, "chain", "current network name (main, test, testnet4, signet, regtest)"},
```

Comment:
Yes, this list of chain names could be de-duplicated as a macro as maflcko suggested or something like a `std::string ListChainTypes()` (perhaps in src/util/chaintypes.h/cpp, src/chainparamsbase.h/cpp, src/common/messages.h/cpp, etc. or where appropriate).